src/twisted/internet/defer.py

```diff
@@ -1573,6 +1573,10 @@ class DeferredSemaphore(_ConcurrencyPrimitive):
     """
 
     def __init__(self, tokens):
+        """
+        @param tokens: initial value of C{tokens} and C{limit}
```

Comment:
[Epytext can resolve `L{something}` in a method docstring to an instance variable `something`](http://epydoc.sourceforge.net/epytext.html) (See section 3.3).  While it's fine to use `C{...}` here, if you switch these to `L{tokens}` and `L{limit}`, pydoctor will generate links to those documented `@ivars`.  That would be pretty neat!